bin/get-server-blocks.php

```diff
@@ -12,6 +12,11 @@
 // with the JSON output.
 error_reporting( 0 );
 
+//avoid direct calls to this file
```

Comment:
This file is only used to generate tests for development and is not in the plugin package.